docs/source/daml/intro/2_Scenario.rst

```diff
@@ -1,82 +1,84 @@
 .. Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 .. SPDX-License-Identifier: Apache-2.0
 
-2 Testing templates using scenarios
-===================================
+2 Testing templates using DAML Script
```

Comment:
Should the filename change accordingly? It's currently still `2_Scenario.rst`.

Reply:
I was wondering about that. We don’t want to break links but we could change it and add a redirect.

Reply:
Renamed and added a redirect.